Mathlib/Algebra/Order/Hom/Basic.lean

```diff
@@ -222,11 +206,11 @@ export AddGroupNormClass (eq_zero_of_map_eq_zero)
 
 export GroupNormClass (eq_one_of_map_eq_zero)
 
-attribute [simp] map_one_eq_zero -- porting note: `to_additive` translation already exists
```

Comment:
What's the story behind this removal?

Reply:
In mathport (https://github.com/leanprover-community/mathlib3port/blob/43c332dc22f5a04dfba2d10ea68e73f56a5df063/Mathbin/Algebra/Order/Hom/Basic.lean#L251) this line is
`attribute [simp, to_additive map_zero] map_one_eq_zero`
However this line with `to_additive` gives the following errors:
```
Basic.lean:210:29
The additive declaration already exists. Please specify this explicitly using `@[to_additive existing]`. [linter.toAdditiveExisting]
Basic.lean:210:17
The translation GroupSeminormClass.map_one_eq_zero ↦ AddGroupSeminormClass.map_zero already exists
```
So `to_additive map_zero` must be removed.

Reply:
Does `attribute [simp, to_additive existing map_zero] map_one_eq_zero` still fail?

Reply:
Yes, it still fails, and just with the `The translation GroupSeminormClass.map_one_eq_zero ↦ AddGroupSeminormClass.map_zero already exists` error message.

Reply:
I think we should keep the porting note: it flags that this is a mistake in mathlib3 that we were forced to fix during porting

Reply:
Right then.